endpoints/osp/osp

```diff
@@ -0,0 +1,582 @@
+#!/usr/bin/env bash
+# -*- mode: sh; indent-tabs-mode: nil; sh-basic-offset: 4 -*-
+# vim: autoindent tabstop=4 shiftwidth=4 expandtab softtabstop=4 filetype=bash
+
+# This script implements the 'osp' endpoint for rickshaw.  It runs 1 or more
+# clients and servers as instances/VMs in an Openstack cluster.
+#
+# Usage:
+# osp [--validate] --endpoint-opts=host=<host>,user=<user>,client:n-m,o-p,server:n-m,o-p 
+#                    --run-id <id> --base-run-dir --image <location>
+#                    --roadblock-server <host> --roadblock-id <id> --roadblock-passwd=<passwd>
+#
+# If --validate is used all options after client/server will be ignored
+#
+
+# Source the base file for common functions and config
+this_endpoint_dir=$(dirname `readlink -e $0` | sed -e 'sX/binXX')
+endpoint_base_dir=$(cd $this_endpoint_dir >/dev/null && cd .. && /bin/pwd)
+if [ -e "$endpoint_base_dir/base" ]; then
+    . "$endpoint_base_dir/base"
+else
+    echo "Could not find endpoint source file "$endpoint_base_dir/base", exiting"
+    exit 1
+fi
+
+echo "#running osp endpoint"
+clear_all_servers="1"
+chroot_rbind_mounts="proc dev sys lib/modules usr/src boot var/run"
+image_cache_size=3
+default_flavor="nfv-large"
+default_availability_zone="nova"
+osruntime="podman"
```

Comment:
should the remotehost and osp endpoints have the same default osruntime?  right now remotehost is chroot and osp is podman...

Reply:
Is it just a problem with sourcing the env file?

Reply:
Yes, I just have not gotten back to see if sourcing it inside the container works

Reply:
Sourcing this file would not work, so reverted back to adding cmdline args to boostrap